ruby-gem/lib/calabash-android/helpers.rb

```diff
@@ -3,7 +3,7 @@
 require 'tempfile'
 require 'escape'
 require 'rbconfig'
-require 'calabash-android/java_keystore'
+require_relative 'java_keystore'
```

Comment:
I don't understand what changed to make this necessary, but it won't 'rake build' without it. Which said, this is technically more correct: it requires that package's own file, not just a general path one.